### DISCUSSION TITLE
Total energy consumption and tire losses

### DISCUSSION THREAD

**Discussion by mariagrazia-tristano** [DLR-VSDC/IEEE-MVC-2023#18] [Q&A] [answered]
Hi,

Looking at the updated Chapter 4C, the total energy loss function depends on the sum over time of p_bat, p_FC and p_loss. However, the Simulink model (specifically the Performance Metrics subsystem) does not contain this last element of the sum but it multiplies Pbat by a negative gain to have it subtracted from p_FC. Are those power losses contained somewhere in this formulation already? While exploring the FMU under controlBus.rexBus one can notice there is a H2PowerLoss included in the VehicleBus that however does not seem to be used in the performance metrics.

Regarding the tire power loss: how are the individual tire power losses computed? If the employed equation is Eq. (6) of the paper, should that not feature the square power of both the longitudinal and lateral component of the force (so the Pythagoras theorem) rather than just their sum?

Many thanks.

**Comment by JBRDLR** [comment]
@tobolar may i ask you to clarify the tire power losses question?

**Comment by tobolar** [comment, marked as answer]
@mariagrazia-tristano 
> Regarding the tire power loss: how are the individual tire power losses computed?

You are absolutely right. There is, unfortunately, a mess in eq. (6). It shall be:
```
P_loss = v_slip * f_slip = v_slip * sqrt(f_long^2 + f_lat^2)
```

Please note, the caltulation in our model (FMU) is indeed done in the right manner.

**Reply by mariagrazia-tristano** [reply]
@tobolar @JBRDLR many thanks. What about the total energy loss function?

**Reply by JBRDLR** [reply]
@imeb can you answer it here directly ?

**Comment by imeb** [comment]
@mariagrazia-tristano

> Looking at the updated Chapter 4C, the total energy loss function depends on the sum over time of p_bat, p_FC and p_loss. However, the Simulink model (specifically the Performance Metrics subsystem) does not contain this last element of the sum but it multiplies Pbat by a negative gain to have it subtracted from p_FC. Are those power losses contained somewhere in this formulation already? While exploring the FMU under controlBus.rexBus one can notice there is a H2PowerLoss included in the VehicleBus that however does not seem to be used in the performance metrics.

 The negative sign after P_bat is for having a consistent sign convention: when the battery is being **discharged**, we would like P_bat to be **positive**. 
Total energy losses "E_loss" along with summed battery and fuel cell energies are brought into the Matlab workspace, and then they are summed together in the script rather than in Simulink model. Please refer to "Reporting/Results.m" lines:
            J_eng       = string(sprintf('%.8f',out.scores(end,1)*2.7778e-7));          % [kWh] Energy performance metric (battery and fuel cell)
            J_loss      = string(sprintf('%.8f',out.scores(end,7)*2.7778e-7));          % [kWh] Total loss performance metric
            **J_eng_tot**   = string(sprintf('%.8f',(out.scores(end,1)+out.scores(end,7))*2.7778e-7));  % [kWh] Total Energy performance metric 

Hope it clarifies your doubts, if not please reply to this message.
Thanks.
